packages/@n8n/nodes-langchain/package.json

```diff
@@ -158,12 +158,12 @@
     "html-to-text": "9.0.5",
     "jest-mock-extended": "^3.0.4",
     "json-schema-to-zod": "2.0.14",
-    "langchain": "0.1.25",
+    "langchain": "0.1.36",
     "lodash": "4.17.21",
     "mammoth": "1.6.0",
     "n8n-nodes-base": "workspace:*",
     "n8n-workflow": "workspace:*",
-    "openai": "4.26.1",
+    "openai": "4.32.1",
```

Comment:
the latest seems to be `4.38.5`

Reply:
Since we're pinning the deps to a specific version I went with the exact version that `@langchain/openai` is using to prevent any potential issues with mismatched types or so